src/libbluechi/log/log.c

```diff
@@ -154,16 +154,18 @@ bool bc_log_get_quiet() {
 
 
 int bc_log_init(struct config *config) {
-        // load log settings from configuration
-        const char *target = NULL;
-        const char *quiet = NULL;
+
+        // set default logging settings
+        bc_log_set_level(LOG_LEVEL_INFO);
+        bc_log_set_log_fn(bc_log_to_journald_with_location);
+        bc_log_set_quiet(false);
```

Comment:
Yes, those are the defaults set for the config and should only be set there. The problem is that the config might contain invalid values since we don't do any validation when reading, e.g. parsing the string to a log target. So for example passing in `LogTarget=journal` would result in not logging at all due to the typo (`journald` is expected) in the current code. Therefore, these values are a kind of fallback to ensure it at least keeps working. We could also return an error here instead. But silently skip an invalid value and not log at all doesn't seem desirable.

Reply:
OK